radixdlt/src/main/java/com/radixdlt/consensus/PendingVotes.java

```diff
@@ -31,48 +31,35 @@
  * Manages pending votes for various vertices
  */
 public final class PendingVotes {
-	private final QuorumRequirements quorumRequirements;
 	private final HashMap<Hash, ECDSASignatures> pendingVotes = new HashMap<>();

```

Comment:
Similarly to PacemakerImpl, could be using interfaces here, and reducing amount of casting in insertVote.